sormas-base/setup/s2s-import-to-truststore.sh

```diff
@@ -97,20 +94,38 @@ while [[ -z "${SORMAS_S2S_TRUSTSTORE_PASS}" ]] || [[ ${#SORMAS_S2S_TRUSTSTORE_PA
   echo
 done
 
-read -p "Please provide the file name of the certificate to import. It should be located inside the sormas2sormas folder: " CRT_FILE_NAME
-CRT_FILE=${SORMAS2SORMAS_DIR}/${CRT_FILE_NAME}
-while [[ -z "${CRT_FILE_NAME}" ]] || [ ! -f "${CRT_FILE}" ]; do
-  echo "File not found in ${SORMAS2SORMAS_DIR} folder."
-  read -p "Please provide the file name of the certificate to import. It should be located inside the sormas2sormas folder: " CRT_FILE_NAME
-  CRT_FILE=${SORMAS2SORMAS_DIR}/${CRT_FILE_NAME}
+while [[ -z "${SORMAS_S2S_HOST_NAME}" ]]; do
+  read -p "Please provide the Hostname of the certificate owner: " SORMAS_S2S_HOST_NAME
 done
 
-# get new certificate alias, which is the same as the Common Name (CN)
-ALIAS=$(openssl x509 -noout -subject -nameopt multiline -in "${CRT_FILE}" | sed -n 's/ *commonName *= //p')
+read -p "Please provide the https port of the server (443): " SORMAS_S2S_HTTPS_PORT
```

Comment:
this should optionally be specified be environment variable. When it is 443, it should be ignored.